management-console/src/main/java/org/wildfly/swarm/management/console/runtime/ManagementConsoleConfiguration.java

```diff
@@ -40,11 +35,15 @@ public ManagementConsoleConfiguration() {
 
     @Override
     public List<Archive> getImplicitDeployments(ManagementConsoleFraction fraction) throws Exception {
-        WARArchive war = ArtifactLookup.get()
-                .artifact("org.jboss.as:jboss-as-console:jar:2.8.25.Final:resources", "management-console-ui.war")
```

Comment:
Ideally this would not be hardcode. Take a look at the swagger web app fraction. It Does retrieve the actual version the maven dependencies through some level of indirection (generated properties file)